packages/mdc-tab-indicator/_mixins.scss

```diff
@@ -27,6 +27,12 @@
   }
 }
 
+@mixin mdc-tab-indicator-not-upgraded-active_ {
```

Comment:
The `mdc-tab-indicator-active_` mixin is only called within the mdc-tab-indicator.scss file right? Why does the `mdc-tab-indicator-not-upgraded-active_` mixin called within the mdc-tab.scss file?

Reply:
`mdc-tab-indicator-not-upgraded-active` makes the active tab indicator visible until the tab indicator is upgraded and initialized. I'll make it non-private and rename it while also removing `mdc-tab-indicator-active` as it serves no real purpose.